specification/hybridcloud/resource-manager/Microsoft.HybridCloud/preview/2023-01-01-preview/hybridCloud.json

```diff
@@ -0,0 +1,1003 @@
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "ResourceFilter": {
+      "type": "object",
+      "description": "Remote Cloud Resource filter.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Remote Resource name to filter on."
+        },
+        "type": {
+          "type": "string",
+          "description": "Remote Resource type to filter on."
+        },
+        "region": {
+          "type": "string",
+          "description": "Remote Resource location/region to filter on."
+        },
+        "tag": {
+          "type": "string",
+          "description": "Remote Resource tags to filter on in key:value format."
+        }
+      }
+    },
+    "DiscoveredResources": {
+      "type": "object",
+      "description": "API to discover resources in remote cloud referred by the connector.",
+      "properties": {
+        "resourceFilter": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ResourceFilter",
+          "description": "Remote cloud resource filter."
+        },
+        "discoveredResources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DiscoveredResource"
+          },
+          "x-ms-identifiers": [
+            "id"
+          ],
+          "description": "List of discovered remote cloud resources."
+        }
+      }
+    },
+    "DiscoveredResource": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Remote cloud resource identifier",
+          "readOnly": true
+        },
+        "bodyContentType": {
+          "type": "string",
+          "description": "Content type of the cloud resource body.",
+          "readOnly": true,
+          "enum": [
+            "Xml",
+            "Json"
+          ],
+          "x-ms-enum": {
+            "name": "bodyContentType",
+            "modelAsString": true
+          }
+        },
+        "body": {
+          "type": "string",
+          "description": "Properties of the remote cloud resource.",
+          "readOnly": true
+        }
+      },
+      "description": "Discovered Resource properties."
+    },
+    "ResourceReference": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID."
+        }
+      },
+      "description": "Resource reference properties."
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "readOnly": true,
+      "description": "The current provisioning state.",
+      "enum": [
+        "Succeeded",
+        "Updating",
+        "Deleting",
+        "Failed",
+        "Canceled"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    }
+  },
+  "parameters": {
+    "apiVersionParameter": {
```

Comment:
Was there any special reason you're not just using api-version parameter from common types here, as the linter suggests?